wrapanapi/lenovo.py

```diff
@@ -86,19 +86,19 @@ def version(self):
         return response['appliance']['version']
 
     def list_servers(self):
-        response = self._service_instance("cabinet?status=includestandalone")
-        cabinets = response['cabinetList'][0]
-        nodes_list = cabinets['nodeList']
-
-        inventory = [node['itemInventory'] for node in nodes_list]
+        cabinet_nodes = []
+        chassis_nodes = []
```

Comment:
There's no need to init cabinet_nodes/chassis_nodes up here, since you re-define them inside the for loop and never return them to the caller :)